[2021-12-01 16:47:44,325] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: FrequencyCount.map2 manual__2021-12-01T16:45:33.951777+00:00 [queued]>
[2021-12-01 16:47:44,339] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: FrequencyCount.map2 manual__2021-12-01T16:45:33.951777+00:00 [queued]>
[2021-12-01 16:47:44,339] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 16:47:44,339] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-12-01 16:47:44,339] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 16:47:44,351] {taskinstance.py:1262} INFO - Executing <Task(MapOperator): map2> on 2021-12-01 16:45:33.951777+00:00
[2021-12-01 16:47:44,356] {standard_task_runner.py:52} INFO - Started process 79 to run task
[2021-12-01 16:47:44,359] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FrequencyCount', 'map2', 'manual__2021-12-01T16:45:33.951777+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/mapred.py', '--cfg-path', '/tmp/tmp0dc3u219', '--error-file', '/tmp/tmp6o0yw5wx']
[2021-12-01 16:47:44,359] {standard_task_runner.py:77} INFO - Job 62: Subtask map2
[2021-12-01 16:47:44,442] {logging_mixin.py:109} INFO - Running <TaskInstance: FrequencyCount.map2 manual__2021-12-01T16:45:33.951777+00:00 [running]> on host 1da30e33d42e
[2021-12-01 16:47:44,491] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FrequencyCount
AIRFLOW_CTX_TASK_ID=map2
AIRFLOW_CTX_EXECUTION_DATE=2021-12-01T16:45:33.951777+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-01T16:45:33.951777+00:00
[2021-12-01 16:47:44,723] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=FrequencyCount, task_id=map2, execution_date=20211201T164533, start_date=20211201T164744, end_date=20211201T164744
[2021-12-01 16:47:44,774] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-01 16:47:44,804] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
